client-v2/src/types/EditMode.ts

```diff
@@ -0,0 +1 @@
+export type EditMode = 'fronts' | 'editions';
```

Comment:
we might want to export `fronts` and `editions` as strings to remove the magic strings above?